src/tiled/propertybrowser.cpp

```diff
@@ -61,9 +61,9 @@
 #include "variantpropertymanager.h"
 #include "wangset.h"
 #include "wangcolormodel.h"
+#include "invertycoordinatehelper.h"
 
 #include <QtGroupPropertyManager>
-
```

Comment:
Please leave this empty line in place. The includes below are from Qt, whereas the include above it from the Qt Property Browser solution.